source/schema/schema-management-upgrade.txt

```diff
@@ -0,0 +1,206 @@
+=========================
+Schema Management Upgrade
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The |bi| version 2.11 release introduces significant improvements to
+stored schema management. The new stored-schema format is
+equivalent to :ref:`DRDL <schema-with-drdl-file>` files. These
+improvements introduce some backwards-breaking changes for certain
+|bi-short| configurations. The following sections outline the upgrade
+path for various |bi-short| configurations.
+
+Standalone Reader Changes
+-------------------------
+
+Use the following table to modify your |bi-short| version 2.10 or
+earlier Standalone Reader configuration to be compatible with version
```

Comment:
At the risk of making this even more complicated, the Standalone Reader configuration isn't present in all earlier versions. I think it was version 2.3 when the new schema generation capabilities arrived? Really though I'm not sure it's even worth it to qualify the "version 2.10 or earlier" part.

Reply:
Fair point, but being that the earliest version of published BIC docs is 2.5 which includes standalone mode, I think it's safe to leave as-is.